Blockzilla/URLBar.swift

```diff
@@ -111,6 +111,8 @@ class URLBar: UIView {
             guard oldValue != showToolset else { return }
             isIPadRegularDimensions = showToolset
             activateConstraints(showToolset, shownConstraints: showToolsetConstraints, hiddenConstraints: hideToolsetConstraints)
+            guard UIDevice.current.orientation.isLandscape else { return }
+            showToolset = false
         }
```

Comment:
I tested on iPad and iPhone and works very well. Just to be sure here, I was thinking that maybe we also should check is the device type is iPhone (`UIDevice.current.userInterfaceIdiom == .phone`) since on the iPad we show Toolset also for portrait, just to be sure that it won't impact any other configuration of the url Bar since there are a lot of constraints etc and since we know that for iPad works just fine at the moment. What do you think? Is it possible to impact something else?

Reply:
Thanks Raluca, very good point to also check if the device is an iphone, to avoid possible future problems with the iPads.